config/routes.rb

```diff
@@ -13,4 +13,10 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show] 
   end
+
+  namespace :api, defaults: { format: json } do 
+    resources :posts, only: [ :index ] do
```

Comment:
- [ ] It would be better if we have  API endpoint that adds' a comment to a post. For example; `POST /users/:user_id/posts/:id/comments` or `POST api/users/:user_id/posts/:id/comments`

Reply:
@addod19 ,
Resolved.
![image](https://user-images.githubusercontent.com/10332499/156661797-4bf2e07b-88b6-4556-a387-67c1ac90012d.png)